### ISSUE TITLE
Cannot setup a new account for Shop Manager

### ISSUE THREAD

**Issue by nei** [reactioncommerce/reaction#5238]
I am not sure if that is a bug or not. But I am trying to setup a new account and I cannot finish the process successfully.

So, I am not sure what is the process but I can't finish this one.

## Prerequisites
* [x] Are you running the latest version?
* [x] Are you able to consistently reproduce the issue?
* [x] Did you search the issue queue for existing issue? [Search issues]

## Issue Description
Providing a clear description of the issue will enable a developer to quickly reproduce and understand the issue. Please provide a detailed description including images, animation and/or videos. For client issues, include browser console errors, for server issues please add terminal output and stack traces.

## Steps to Reproduce
1. bring the platform up using the make command.
2. Login on http://localhost:3000/operator/accounts using default admin@localhost
3. Add name and email, choose Shop manager and click on send invitation
4. I receive the email with a button link to http://localhost:3000/account/enroll/A3p2cRuJsqkJjMyAQ
5. The link sends me to the login page (I was expecting something like set password)
6. Since I do not have login, I click on Register, fill the form and I get an error user already exist with the email.
6.1 If I try to login it say ```User has no password set```

## Possible Solution
If that is not a bug I would expect some clarification on how to do it.

### Versions
```
Node: 11.14.0
NPM: 6.9.0
Meteor Node: 8.15.1
Meteor NPM: 6.9.0
Reaction CLI: 0.29.0
Docker: 18.09.2
```

**Comment by loan-laux** [comment]
@nei This is a bug that I fixed recently. The PR was merged right after you filed this issue.

Can you pull the latest release candidate and confirm that it works? Just tried and I can sign up fine.

**Comment by kieckhafer** [comment]
This should be resolved by #5799

**Comment by HarisSpahijaPon** [comment]
Can we close this issue as it is fixed in #5799 for 3.0?

**Comment by aldeed** [comment]
Yes, I think so
